src/components/VideoOverview/VideoOverview.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { getDateDiff, getVideoDate, VideoDate } from '../../helpers/date';
+import { clipTitle } from '../../helpers/titles';
+import VideoEmbed from '../VideoEmbed';
+import { StyledContainer, StyledContent, StyledImg, StyledMeta, StyledPlayIcon, StyledTitle } from './styles';
+import { Video } from './types';
+
+export default function VideoOverview({ item }: { item: Video }) {
+  const [publishedAgo, setPublishedAgo] = useState<VideoDate>();
+  const [loadVideo, setLoadVideo] = useState<boolean>(false);
+
+  useEffect(() => {
+    setPublishedAgo(getVideoDate(getDateDiff(item.snippet.publishedAt)));
```

Comment:
Not sure that this is the right place to do parsing/mapping of the video item. If we can leave the component clear - let's do it. Such kind mapping/parsing should be done in the other layer of the app, the fetching layer.

Reply:
I understand, but I see one downside:  If I hardcode that upon fetch and pass to videos state instead of recalculating each time a Video is loaded this information is not accurate as seconds, minutes or hours might change. However I see that putting it in [] dep array does not make a difference in this case